cmd/client/cmd/root.go

```diff
@@ -92,7 +99,11 @@ func GetRootCmd(args []string, printf, fatalf shared.FormatFn) *cobra.Command {
 	rootCmd.PersistentFlags().StringVarP(&rootArgs.mixerAddress, "mixer", "m", "localhost:9091",
 		"Address and port of a running Mixer instance")
 	rootCmd.PersistentFlags().IntVarP(&rootArgs.repeat, "repeat", "r", 1,
-		"Sends the specified number of requests in quick succession")
+		"Sends the specified number of requests in quick succession. Concurency flag limits the number of outstanding requests.")
+	rootCmd.PersistentFlags().IntVarP(&rootArgs.concurrency, "concurrency", "c", 1,
```

Comment:
If the flag is defined here, then the behavior should also apply to the Report command. Right now, since it's only implemented in Check, this flag and the next should be applied to the Check command only.